src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -821,7 +821,9 @@ void EQLWindowSurfaceHeatBalance(EnergyPlusData &state,
     ConvHeatGainWindow = state.dataSurface->Surface(SurfNum).Area * HcIn * (SurfInsideTemp - TaIn);
     // Window heat gain (or loss) is calculated here
     state.dataSurface->SurfWinHeatGain(SurfNum) =
-        state.dataSurface->SurfWinTransSolar(SurfNum) + ConvHeatGainWindow + NetIRHeatGainWindow + ConvHeatFlowNatural;
+        state.dataSurface->SurfWinTransSolar(SurfNum) + ConvHeatGainWindow + NetIRHeatGainWindow + ConvHeatFlowNatural -
+        (state.dataSurface->SurfWinLossSWZoneToOutWinRep(SurfNum) +
+         state.dataHeatBalSurf->SurfWinInitialDifSolInTrans(SurfNum) * state.dataSurface->Surface(SurfNum).Area);
```

Comment:
Subtract missing solar leaving components from `SurfWinHeatGain` for equivalent layer windows.